arb_os/gasAccounting.mini

```diff
@@ -115,11 +276,16 @@ public impure func gasAccounting_startTxCharges(
     return Some(());
 }
 
-public impure func gasAccounting_endTxCharges() -> option<GasUsage> {
+public impure func gasAccounting_endTxCharges(storageDelta: int) -> option<GasUsage> {
```

Comment:
Charging for the cost of storage after the whole transaction has completed seems potentially dangerous since it means that nested call can't have it's gas usage limited by the amount of gas passed to the call unless I'm missing something here

Reply:
Let's discuss this one.